src/components/PrepareSubmitPage.jsx

```diff
@@ -48,6 +71,11 @@ export function PrepareSubmitPage({ form, submission }) {
         <Normaltekst className="margin-bottom-default">
           Du trenger pdf-filen i neste steg. Kom deretter tilbake hit for å gå videre til innsending av søknaden.
         </Normaltekst>
+        <div>
+          <button className="knapp knapp--hoved" onClick={lastNedFoersteside}>
```

Comment:
det bør være usynlig for bruker på master, kan bruke environment variable til å skille ut

Reply:
Viser nå knappen bare i development.